ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -786,11 +786,10 @@ KEXT_STATIC int HandleFileOpOperation(
         if (0 != toErr)
         {
             KextLog_Error("HandleFileOpOperation (KAUTH_FILEOP_RENAME): vnode_lookup failed, errno %d for path '%s'", toErr, newPath);
-            goto CleanupAndReturn;
```

Comment:
Make these same change for KAUTH_FILEOP_LINK (see line 859 below)

Reply:
I'm still iterating on this PR (hence draft status) - I'll ping you via the "request review" button when I'm done so you don't end up reviewing draft code.

Reply:
Sure, sorry about missing that and jumping the gun!

Reply:
No worries, just pointing it out so you don't end up wasting time! Github's "Draft" indicator is a bit too subtle I think, I'll try to remember to add a [WIP] or similar to the title again in future.